tools/wpt/browser.py

```diff
@@ -63,6 +63,18 @@ class Browser(object):
     def __init__(self, logger):
         self.logger = logger
 
+    def _get_dest(self, dest, channel):
+        if dest is None:
+            # os.getcwd() doesn't include the venv path
+            dest = os.path.join(os.getcwd(), "_venv")
```

Comment:
More for my own info - isn't the _venv dir suffixed with the py version these days? Curious why it doesn't care about that here.

Reply:
Yeah it is, and AFAICT, `dest` is never `None` when executed from `wpt run` or `wpt install`, but I'll leave it alone for now until I do some further investigation for a cleanup/refactoring.